pages/index.js

```diff
@@ -11,21 +13,27 @@ export default function HomePage() {
       </Head>
 
       <StyledTitle>All Outivities</StyledTitle>
-
       <main>
         <article>
           <StyledOutivityCardContainer>
             {outivities.map((outivity) => (
               <StyledOutivityCard key={outivity.id}>
-                <StyledImage
-                  src={outivity.image}
-                  alt={outivity.title}
-                  width={300}
-                  height={200}
-                />
+                <Link
+                  href={{
+                    pathname: `[slug]`,
+                    query: { slug: outivity.slug },
+                  }}
+                >
```

Comment:
If the route is straightforward and doesn't require any additional query parameters, you can use a simple string for the href.


```suggestion
              <Link href={`/${outivity.slug}`}>
```

Reply:
We actually tried to use a simple string in the beginning.. which didn't work for some reasons -> probably because of a simpel misspelling 🤔 

But now it's working – thank you 🙌